ensawards.org/src/data/bestPractices.ts

```diff
@@ -110,24 +107,24 @@ export const BEST_PRACTICES: BestPractice[] = [
 ];
 
 export const BEST_PRACTICE_CATEGORIES: BestPracticeCategory[] = [
-  {
-    id: "forward-resolution",
-    slug: "forward-resolution",
-    name: "Forward Resolution",
-    description:
-      "Lookup social records, the contenthash of decentralized websites, avatar images, multichain deposit addresses, and more.",
-    status: CategoryStatus.Updated,
-    bestPractices: [recognizeAllENSNames],
-  },
   {
     id: "contract-naming",
     slug: "contract-naming",
     name: "Contract naming",
     description:
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+      "Assign and display ENS names for smart contracts to improve security and enhance UX.",
     status: CategoryStatus.Updated,
     bestPractices: [nameYourSmartContracts, displayNamedSmartContracts],
   },
+  {
+    id: "forward-resolution",
+    slug: "forward-resolution",
+    name: "Forward Resolution",
+    description:
+      "Lookup social records, the contenthash of decentralized websites, avatar images, multichain deposit addresses, and more.",
```

Comment:
```suggestion
      "Lookup the onchain addresses, avatar image, social records, and decentralized website for an ENS name.",
```

Strategy: Order use cases by their frequency of use in the market. This makes it more "punchy".